src/borg/archiver.py

```diff
@@ -3370,6 +3373,8 @@ def define_borg_mount(parser):
         subparser.set_defaults(func=self.do_delete)
         subparser.add_argument('-n', '--dry-run', dest='dry_run', action='store_true',
                                help='do not change repository')
+        subparser.add_argument('--consider-checkpoints', action='store_true', dest='consider_checkpoints',
```

Comment:
why is that useful for borg delete?

Reply:
delete and info command use the list command, so that's why I added that arg. but as you suggest it's not useful, now I forced the arg "consider-checkpoints" to true inside the two commands. I don't know if it's the right way to do that